pkgs/development/embedded/blisp/default.nix

```diff
@@ -31,12 +36,16 @@ stdenv.mkDerivation {
     "-DBLISP_USE_SYSTEM_LIBRARIES=ON"
   ];
 
+  passthru.tests.version = testers.testVersion {
+    package = blisp;
```

Comment:
If you use the finalAttrs pattern for stdenv you dont need to use the package name variable.

See this for an example: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/tilemaker/default.nix#L36